### PR TITLE
refactor: updates to latest Gravity release

### PR DESCRIPTION
**Pull request by james-nash**
Bumps to the latest Gravity NEXT pre-release, which introduces new and updated multi-column layout
classes. This change also updates some of the website templates' mark-up to use those new classes
correctly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,9 +56,9 @@
     "npm": ">= 5.8.0"
   },
   "dependencies": {
+    "@buildit/gravity-ui-web": "^3.0.0-alpha.6",
     "@buildit/job-listings": "^1.1.0",
-    "@buildit/gravity-ui-web": "next",
```
Comment (dw-buildit) [inline package.json]: shouldn't this remain as `next`? Just the `package-lock.json` will change to a specific alpha.6 version?

Comment (james-nash) [reply package.json]: Ah yes, you're right. Will fix that.

**Comment by james-nash** [comment]
@dw-buildit I fixed the `package.json` issue you spotted.

I also noticed a layout problem caused by our new column layout mixin: The story list's negative margins make it wider than the viewport (even if the positioning makes it _look_ like it doesn't). So, on smaller viewports this caused the page to be wider than the viewport and you got horrible horizontal scrolling.

It needs to be fixed upstream in Gravity but might be non-trivial, since we'll need to find a different way of doing the column layouts. I'll raise an issue for this in a moment.

But, to not hold up website stuff in the short term, I added a small band-aid in the website's SASS (`overflow-x: hidden;` on the `.grav-o-container`), to work around this.

File: src/sass/style.scss [hunk 2]
```diff
@@ -44,3 +44,9 @@
 /* Trumps           */
 /********************/
 
+// Temporary work-around for bug in Gravity's
+// column layout styling that causes the page to
+// overflow the viewport horizontally.
+.grav-o-container {
+  overflow-x: hidden;
```
Comment (dw-buildit) [inline src/sass/style.scss]: This isn't lovely 😆 

Not going to block the merge for it though

Comment (james-nash) [reply src/sass/style.scss]: I know - far from it.

Thanks for approving though!

**Pull request merged**